CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [0.5.0] 2019-07-17
```

Comment:
You also need to change https://github.com/giantswarm/kubernetes-node-exporter/blob/eb7b687f36463e1508c6f39614bec715bfe3bb58/helm/kubernetes-node-exporter-chart/Chart.yaml#L5

Reply:
changed but not commited